src/App.js

```diff
@@ -1,5 +1,41 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "../src/constants/messages.js";
+import carInputValidation from "./utils/carInputValidation.js";
+import countInputValidation from "./utils/countInputValidation.js";
+import progress from "./utils/progress.js";
+import racePrint from "./utils/racePrint.js";
+import winner from "./utils/winner.js";
 class App {
-  async play() {}
+  async play() {
+    MissionUtils.Console.print(MESSAGES.START);
+    const CARS_NAMES = await MissionUtils.Console.readLineAsync("");
+    carInputValidation(CARS_NAMES);
+
+    MissionUtils.Console.print(MESSAGES.ANSWER_COUNT);
+    const COUNT = await MissionUtils.Console.readLineAsync("");
```

Comment:
```suggestion
    const COUNT = await MissionUtils.Console.readLineAsync(MESSAGES.ANSWER_COUNT);
```
여기도 이렇게 축약해볼 수 있을 것 같아요!